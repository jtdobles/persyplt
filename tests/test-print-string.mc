void main(){
	prints('Hello World!');

	//this comment will be ignored

	string a = 'PLT is hard';
	prints(a);
}
