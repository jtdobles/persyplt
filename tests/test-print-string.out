'Hello World!'
'PLT is hard'
